server/src/test/java/org/opensearch/action/support/master/TransportMasterNodeActionTests.java

```diff
@@ -506,7 +512,8 @@ public void testMasterFailoverAfterStepDown() throws ExecutionException, Interru
         new Action("internal:testAction", transportService, clusterService, threadPool) {
             @Override
             protected void masterOperation(Request request, ClusterState state, ActionListener<Response> listener) throws Exception {
```

Comment:
`masterOperation` -> `clusterManagerOperation`

Reply:
I will rename this method in the next phase.
Although the method `masterOperation` is not public, but its abstract class `TransportMasterNodeAction` is public (https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/server/src/main/java/org/opensearch/action/support/master/TransportMasterNodeAction.java#L72), so I will change it after completing the internal usages in this PR.